### PR TITLE
#160617643 Users should be able to share articles on social media and via email

### PR DESCRIPTION
**Pull request by AmosWels**
#### What does this PR do?
- Allows Users to share articles on social media and via email
#### Description of Task to be completed?
- Add a feature to allow sharing by email 
- Enable sharing an article through social media
#### How should this be manually tested?
- Visit the application at [ah haven](https://ah-frontend-lannister-pr-23.herokuapp.com/) 
##### Share article page
image
#### Share by mail
image
#### Share by GooglePlus
image
#### Share by Facebook
image
#### Share by Twitter
image
#### Share by LinkedIn
image
#### Share by Whatsapp
image
#### What are the relevant pivotal tracker stories?
[#160617643](https://www.pivotaltracker.com/story/show/160617643)

### CODE REVIEW

File: src/assets/articleAssets/demo2.scss [hunk 1]
```diff
@@ -467,4 +468,34 @@ a:hover {
...
+  background-color: rgb(216, 212, 212) !important;
+}
+
+.emailIcon {
+  background-color: transparent !important;
+  margin-left: 2px;
+  font-size: 30px;
+}  
+  
+
```
Comment (LehruAngela) [inline src/assets/articleAssets/demo2.scss:501]: Leave one line at the end of file

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/__tests__/articleTests/viewArticle.test.js [hunk 2]
```diff
@@ -54,3 +55,14 @@ it("viewing article", () => {
...
+it("email share component", () => {
+  let dispatch = jest.fn();
+  let handleEmail = jest.fn();
+  let props = {
+    dispatch,
+    handleEmail
+  };
+  let component = shallow(<EmailModal {...props} />);
+  component;
+});
```
Comment (LehruAngela) [inline src/__tests__/articleTests/viewArticle.test.js:72]: What does calling component do?

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/__tests__/articleTests/viewArticle.test.js (hunk 2 above)
Comment (AmosWels) [reply src/__tests__/articleTests/viewArticle.test.js:72]: I was using component as a variable for holding the particular component that I render for testing.

File: src/assets/articleAssets/demo2.scss (hunk 1 above)
Comment (AmosWels) [reply src/assets/articleAssets/demo2.scss:501]: I have added the new line

File: src/__tests__/articleTests/viewArticle.test.js (hunk 2 above)
Comment (LehruAngela) [reply src/__tests__/articleTests/viewArticle.test.js:72]: Do you `expect` anything to happen. When you're writing tests, you often need to check that values meet certain conditions using expect

**Review by LehruAngela — CHANGES_REQUESTED** [review]

**Pull request merged**